scripts/data-retention-remove-contact-information-data.py

```diff
@@ -53,10 +53,7 @@
         auth_token=get_auth_token('api', stage)
     )
 
-    users = sorted(
-        data_api_client.find_users_iter(role='supplier', personal_data_removed=True),
-        key=lambda i: i['supplier']['supplierId']
-    )
+    users = sorted(data_api_client.find_users_iter(role='supplier'), key=lambda i: i['supplier']['supplierId'])
```

Comment:
Why sort them here when they're grouped by supplier id in the next line?

Reply:
One of my least favourite Python things. Groupby doesn't group unless they're already in groups 🤔